vnc.html

```diff
@@ -286,6 +286,10 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br>VNC</h1>
                 <label>Password:</label>
                 <input id="noVNC_password_input" type="password">
             </li>
+            <li id="noVNC_fingerprint_block">
+                <label>Fingerprint:</label>
+                <div id="noVNC_fingerprint"></div>
+            </li>
```

Comment:
There needs to be an option to approve or reject this key. And there should probably be a more descriptive text what this question means.